azure-pipelines.yml

```diff
@@ -55,599 +55,591 @@ resources:
-
-      - task: JFrogNuGet@1
-        displayName: 'Publish SonarAnalyzer.CFG NuGet on repox'
-        inputs:
-          command: 'push'
-          artifactoryConnection: 'repox_qa_deployer_token'
-          targetDeployRepo: '$(ARTIFACTORY_NUGET_REPO)'
-          module: SonarAnalyzer.CFG
-          pathToNupkg: '$(Build.ArtifactStagingDirectory)/packages/SonarAnalyzer.CFG.*.nupkg'
-          collectBuildInfo: true
-          buildName: sonar-dotnet_nuget
-          buildNumber: '$(Build.BuildId)'
-
-      - task: JFrogPublishBuildInfo@1
-        inputs:
-          artifactoryConnection: 'repox_qa_deployer_token'
-          buildName: sonar-dotnet_nuget
-          buildNumber: '$(Build.BuildId)'
-
-      - task: PublishPipelineArtifact@1
-        displayName: 'Publish analyzer binaries as pipeline artifact'
-        inputs:
-          path: analyzers/packaging/binaries/
-          artifact: Binaries
-
-      - task: PublishPipelineArtifact@1
-        displayName: 'Publish analyzer test binaries as pipeline artifact'
-        inputs:
-          path: '$(UnitTestProjectPath)\bin'
-          artifact: TestBinaries
-
-- template: stage-with-burgr-notifications.yml@pipelines-yaml-templates
-  parameters:
-    burgrName: 'qa'
-    burgrType: 'qa'
-    stageName: 'qa'
-    # .NET code analysis, UTs, ITs, build Java and publish SC QG
-    stageDisplayName: 'Tests:'
-    stageDependencies: build
-
-    jobs:
-    - job: runUnitTestsDotNet
-      displayName: '.NET UTs'
-      workspace:
-        clean: all
-      strategy:
-        matrix:
-          Net48:
-            CoverageArtifactName: 'DotNetCoverageNet48'
-            TestResultsArtifactName: 'DotNetTestResultsNet48'
-            ProjectFilePath: 'tests\SonarAnalyzer.Test\SonarAnalyzer.Test.csproj'
-            FrameworkMoniker: 'net48'
-          Net80:
-            CoverageArtifactName: 'DotNetCoverageNet8'
-            TestResultsArtifactName: 'DotNetTestResultsNet8'
-            ProjectFilePath: 'tests\SonarAnalyzer.Test\SonarAnalyzer.Test.csproj'
-            FrameworkMoniker: 'net8.0'
-          Styling:
-            CoverageArtifactName: 'DotNetCoverageStyling'
-            TestResultsArtifactName: 'DotNetTestResultsStyling'
-            ProjectFilePath: 'tests\SonarAnalyzer.CSharp.Styling.Test\SonarAnalyzer.CSharp.Styling.Test.csproj'
-            FrameworkMoniker: 'net8.0'
-          TestFramework:
-            CoverageArtifactName: 'DotNetCoverageTestFramework'
-            TestResultsArtifactName: 'DotNetTestResultsTestFramework'
-            ProjectFilePath: 'tests\SonarAnalyzer.TestFramework.Test\SonarAnalyzer.TestFramework.Test.csproj'
-            FrameworkMoniker: 'net8.0'
-          ITs.JsonParser:
-            CoverageArtifactName: 'ITs.JsonParserCoverage'
-            TestResultsArtifactName: 'ITs.JsonParserResults'
-            ProjectFilePath: 'tests\ITs.JsonParser.Test\ITs.JsonParser.Test.csproj'
-            FrameworkMoniker: 'net8.0'
+- stage: build
```

Comment:
`Artifacts:` stage contains three `Nofity burgr` jobs.

You will need to create a PR against Azure DevOps and update `promote-stage.yml`, create a new release tag there, and then use it in this pipeline.

Probably as s major version bump